### ISSUE TITLE
Not able to pass a function into columnWidth

### ISSUE THREAD

**Issue by DanielRamosAcosta** [techniq/mui-virtualized-table#12]
The docs state that `columnWidth` could either be a `number` or a `func`. But when giving the prop a function it throws the following error:

```
checkPropTypes.js:19 Warning: Failed prop type: Invalid prop `columnWidth` of type `function` supplied to `MuiTable`, expected `number`.
    in MuiTable (created by WithStyles(MuiTable))
    in WithStyles(MuiTable) (created by AutoSizer)
(...)
Uncaught Error: Invalid size returned for cell 0 of value NaN
    at CellSizeAndPositionManager.getSizeAndPositionOfCell (CellSizeAndPositionManager.js:105)
    at CellSizeAndPositionManager._binarySearch (CellSizeAndPositionManager.js:252)
    at CellSizeAndPositionManager._findNearestCell (CellSizeAndPositionManager.js:305)
    at CellSizeAndPositionManager.getVisibleCellRange (CellSizeAndPositionManager.js:217)
    at ScalingCellSizeAndPositionManager.getVisibleCellRange (ScalingCellSizeAndPositionManager.js:160)
    at Grid._calculateChildrenToRender (Grid.js:811)
    at Grid.render (Grid.js:717)
    at finishClassComponent (react-dom.development.js:13194)
    at updateClassComponent (react-dom.development.js:13156)
    at beginWork (react-dom.development.js:13825)
(...)
Uncaught Error: Invalid size returned for cell 0 of value NaN
    at CellSizeAndPositionManager.getSizeAndPositionOfCell (CellSizeAndPositionManager.js:105)
    at CellSizeAndPositionManager._binarySearch (CellSizeAndPositionManager.js:252)
    at CellSizeAndPositionManager._findNearestCell (CellSizeAndPositionManager.js:305)
    at CellSizeAndPositionManager.getVisibleCellRange (CellSizeAndPositionManager.js:217)
    at ScalingCellSizeAndPositionManager.getVisibleCellRange (ScalingCellSizeAndPositionManager.js:160)
    at Grid._calculateChildrenToRender (Grid.js:811)
    at Grid.render (Grid.js:717)
    at finishClassComponent (react-dom.development.js:13194)
    at updateClassComponent (react-dom.development.js:13156)
    at beginWork (react-dom.development.js:13825)
(...)
react-dom.development.js:14227 The above error occurred in the <Grid> component:
    in Grid (created by MultiGrid)
    in div (created by MultiGrid)
    in div (created by MultiGrid)
    in MultiGrid (created by MuiTable)
    in div (created by Table)
    in Table (created by WithStyles(Table))
    in WithStyles(Table) (created by MuiTable)
    in MuiTable (created by WithStyles(MuiTable))
    in WithStyles(MuiTable) (created by AutoSizer)
    in div (created by AutoSizer)
```

**Comment by DanielRamosAcosta** [comment]
Thanks for the fix!

**Comment by techniq** [comment]
Thanks to @alexander7161.  Just published as `2.1.0`
